src/typeorm-aurora-data-api-driver.ts

```diff
@@ -1,95 +1,113 @@
 // @ts-ignore
 import createDataApiClient from 'data-api-client'
 
-const quoteCharacters = ['\'', '"']
-
-export default class DataApiDriver {
-  public static transformQueryAndParameters(query: string, parameters?: any[]): any {
-    let numberOfParametersInQueryString = 0
-
-    let newQueryString = ''
-    let currentQuote = null
-
-    for (let i = 0; i < query.length; i += 1) {
-      const currentCharacter = query[i]
-      const currentCharacterEscaped = i !== 0 && query[i - 1] === '\\'
-
-      if (currentCharacter === '?' && !currentQuote) {
+const transformQuery = (query: string, parameters?: any[]): [string, number] => {
```

Comment:
Guess if we extract these functions they can live in something like query-transfromation.utils.ts or something similar. Nice job making the class cleaner

Reply:
Yeah sounds like a good plan. I will leave that as an exercise for later.